docs/troubleshooting.mdx

```diff
@@ -9,29 +9,32 @@ import TabItem from '@theme/TabItem';
 # Troubleshooting
 ## Panel errors
 
-If you see a `500 | SERVER ERROR` or `An unexpected error was encountered while processing this request` you have to check your panel logs by running the following command.
-
+* `502 Bad Gateway Error` or some code in plain text you either didn't install php-fpm or set the wrong version in your webserver vhost.
+* `500 | SERVER ERROR` or `An unexpected error was encountered while processing this request` you have to check your panel logs by running the following command.
+* `CSRF token mismatch` you are most likely using HTTP instead of HTTPS in `APP_URL` or you didn't force `SESSION_SECURE_COOKIE` to false while using HTTP.
```

Comment:
Please move the 502 error and CSRF error to the "common errors" section. The introductory text should stay.